ios/FluentUI/Navigation/SearchBar/SearchBarTokenSet.swift

```diff
@@ -176,7 +180,8 @@ extension SearchBarTokenSet {
 public extension SearchBar {
     @objc(MSFSearchBarStyle)
     enum Style: Int {
-        case lightContent
-        case darkContent
+        case canvas
+        case hosted
+        case brand
```

Comment:
another thought on reducing the api breaking is to keep the lightcontent and darkcontent and the new 3 styles are addtion. the old one kind of map to the new one. we should still have deprecation warning documenation but it does ease moving people to new style api